huaweicloud/services/cce/resource_huaweicloud_cce_node_v3.go

```diff
@@ -947,9 +966,48 @@ func resourceCCENodeV3Update(ctx context.Context, d *schema.ResourceData, meta i
 		}
 	}
 
+	if d.HasChange("auto_renew") {
+		bssClient, err := config.BssV2Client(config.GetRegion(d))
+		if err != nil {
+			return diag.Errorf("error creating BSS V2 client: %s", err)
+		}
+		ecsId, err := getUnderlyingResourceId(nodeClient, bssClient, d.Get("cluster_id").(string), d.Id())
```

Comment:
can you try to use `server_id` attribute?

Reply:
it's workful